index.html

```diff
@@ -1,188 +1,267 @@
+            <li>html</li>
+          </ul>
+          <button type="button">See Project</button>
+        </div>
+      </div>
+      <div class="portfolio-card">
+        <div class="card-body">
+          <h3>Multi-Post Stories Gain+Glory</h3>
+          <ul>
+            <li>Ruby on rails</li>
+            <li>css</li>
+            <li>Javascript</li>
+            <li>html</li>
+          </ul>
+          <button type="button">See Project</button>
+        </div>
+      </div>
+    </section>
+    <section class="About">
+      <div class="about-headline">
+        <div class="section-heading">
+          <h2>About me</h2>
+          <p class="desc">
+            I can help you build a product, feature or website Look through some
+            of my work and experience! If you like what you see and have a
+            project you need coded, don't hestiate to contact me.
+          </p>
+          <button type="button">Get My Resume</button>
+        </div>
+      </div>
+      <ul>
+        <li>
+          <img src="img/IconLangguage.svg" alt="Language" />
+          <h3>Languages</h3>
+          <ul>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>html</li>
+            <li>CSS</li>
+          </ul>
+        </li>
+        <li>
+          <img src="img/IconFramework.svg" alt="Language" />
+          <h3>Frameworks</h3>
+          <ul>
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSpec</li>
+            <li>Capybara</li>
+            <li>Selenium</li>
+          </ul>
+        </li>
+        <li>
+          <img src="img/IconSkills.svg" alt="Language" />
+          <h3>Skills</h3>
+          <ul>
+            <li>Codekit</li>
+            <li>GitHub</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </li>
+      </ul>
+    </section>
+    <section class="footer">
+      <div class="form-desc">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </div>
+      <form
+        action="https://formspree.io/f/xqknezwl "
+        class="contact-form"
+        method="post"
+      >
+        <ul>
+          <li>
+            <input
+              type="text"
+              name="Full_name"
+              placeholder="Full Name"
+              maxlength="30"
+              required
+            />
+          </li>
+          <li>
+            <input
+              type="email"
+              name="email"
+              placeholder="Saadat@gmail.com"
+              required
+            />
+          </li>
+          <li>
+            <textarea
+              name="message"
+              id="msg"
+              placeholder="Enter your message"
+              maxlength="500"
+              required
+            ></textarea>
+          </li>
+          <li>
+            <button type="submit">Get in touch</button>
+          </li>
+        </ul>
```

Comment:
Please try to follow the details on the figma's design 📝 Your inputs for name, email, and textarea have a red border. What do you think if we change those borders to meet the project requirements 🙌😎

#### Your design 
![image](https://user-images.githubusercontent.com/15807343/149045473-e8b1a649-80f9-4c8a-82ce-a2e1b5702df0.png)

#### Figma's design 
![image](https://user-images.githubusercontent.com/15807343/149045533-aa3eb12e-f471-4e80-9938-c2e8aa528d8c.png)